cmd/srcd/cmd/parse.go

```diff
@@ -123,6 +110,12 @@ func (cmd *parseUASTCmd) Execute(args []string) error {
 		}
 
 		if err != nil {
+			if strings.Contains(err.Error(), "missing driver for language") {
```

Comment:
do we have an issue in bblfsh to export this error instead of `strings.Contains`?

Reply:
yes, it's this one https://github.com/src-d/engine/issues/297

Reply:
wait. this is an issue in engine about aliasing.

I'm talking about an issue in bblfsh-sdk / bblfsh-go-client that would differentiate lang not found error from any other error.
I don't see any link or any conversation about it in 297.

Reply:
@carlosms ping